ledger/accountdb.go

```diff
@@ -764,14 +1059,54 @@ func (ba *baseAccountData) GetAccountData() basics.AccountData {
 	}
 }
 
+func (ba *baseAccountData) GetCoreAccountData() ledgercore.AccountData {
```

Comment:
Duplicate of above GetLedgerCoreAccountData()

Reply:
thanks.